openfisca_core/tests/dummy_country/model/model.py

```diff
@@ -0,0 +1,170 @@
+
+
+class dom_tom(Variable):
+    column = BoolCol
+    entity = Famille
+    label = u"La famille habite-t-elle les DOM-TOM ?"
+
+    def function(famille, period):
+        period = period.start.period(u'year').offset('first-of')
+
+        depcom = famille('depcom', period)
+
+        return period, np.logical_or(startswith(depcom, '97'), startswith(depcom, '98'))
+
+
+class revenu_disponible(Variable):
+    column = FloatCol
+    entity = Individu
+    label = u"Revenu disponible de l'individu"
+
+    def function(individu, period, legislation):
+        period = period.start.period(u'year').offset('first-of')
+        rsa = individu('rsa', period, options = [ADD])
+        salaire_imposable = individu('salaire_imposable', period)
+        taux = legislation(period).impot.taux
+
+        return period, rsa + salaire_imposable * (1 - taux)
+
+
+class revenu_disponible_famille(Variable):
+    column = FloatCol
+    entity = Famille
+    label = u"Revenu disponible de la famille"
+
+    def function(famille, period):
+        revenu_disponible = famille.members('revenu_disponible', period)
+        return period, famille.sum(revenu_disponible)
+
+
+class rsa(DatedVariable):
+    column = FloatCol
+    entity = Individu
+    label = u"RSA"
+
+    @dated_function(datetime.date(2010, 1, 1))
+    def function_2010(individu, period):
+        period = period.start.period(u'month').offset('first-of')
+        salaire_imposable = individu('salaire_imposable', period, options = [DIVIDE])
+
+        return period, (salaire_imposable < 500) * 100.0
+
+    @dated_function(datetime.date(2011, 1, 1), datetime.date(2012, 12, 31))
+    def function_2011_2012(individu, period):
+        period = period.start.period(u'month').offset('first-of')
+        salaire_imposable = individu('salaire_imposable', period, options = [DIVIDE])
+
+        return period, (salaire_imposable < 500) * 200.0
+
+    @dated_function(datetime.date(2013, 1, 1))
+    def function_2013(individu, period):
+        period = period.start.period(u'month').offset('first-of')
+        salaire_imposable = individu('salaire_imposable', period, options = [DIVIDE])
+
+        return period, (salaire_imposable < 500) * 300
+
+
+class salaire_imposable(Variable):
+    column = FloatCol
+    entity = Individu
+    label = u"Salaire imposable"
+
+    def function(individu, period):
+        period = period.start.period(u'year').offset('first-of')
+        dom_tom = individu.famille('dom_tom', period)
+
+        salaire_net = individu('salaire_net', period)
+
+        return period, salaire_net * 0.9 - 100 * dom_tom
+
+
+class salaire_net(Variable):
+    column = FloatCol
+    entity = Individu
+    label = u"Salaire net"
+
+    def function(individu, period):
+        period = period.start.period(u'year').offset('first-of')
+        salaire_brut = individu('salaire_brut', period)
+
+        return period, salaire_brut * 0.8
+
+
+class csg(Variable):
```

Comment:
It seems to be used in the tests, and re-designing the tests is out of the scope of this PR ;)